asciidoc/modules/con_mg_changes-in-json.adoc

```diff
@@ -78,3 +78,7 @@ Use the following example code to convert a Base64URL to Base64 encoder.
 String base64url = someJsonObject.getString("base64encodedElement")
 String base64 = Json.toBase64(base64url);
 ----
+
+== Removed the JSON converter method from trust options
+
+The `TrustOptions.toJSON` method has been removed.XXX
```

Comment:
@vietj Should we add more details for this note.

Original note:
https://github.com/vert-x3/wiki/wiki/4.0.0-Deprecations-and-breaking-changes#%EF%B8%8F-remove-trustoptions-tojson-method

Reply:
No, it was just removed because there was no interest for it and it was creating some friction with 4.0 improvements.